pystarport/cluster.py

```diff
@@ -902,9 +903,9 @@ def relayer_chain_config(data_dir, chain, relayer_chains_config):
 
 
 def init_cluster(
-    data_dir, config_path, base_port, image=IMAGE, cmd=None, gen_compose_file=False
+    data_dir, config_path, base_port, dotenv, image=IMAGE, cmd=None, gen_compose_file=False
```

Comment:
```suggestion
    data_dir, config_path, base_port, dotenv=None, image=IMAGE, cmd=None, gen_compose_file=False
```

to make it backward compatible.